test/benchmark/l4bench.go

```diff
@@ -0,0 +1,194 @@
+package benchmarktest
```

Comment:
Typo in commit - allowsjus --> allows us

Reply:
yep thanks.